react-chart.js.org

tannerlinsley opened the pull request:
- [x] There is reasonable content on the page (see: [No Content](https://github.com/js-org/dns.js.org/wiki/No-Content))
- [x] I have read and accepted the [ToS](http://dns.js.org/terms.html)

indus commented:
I've already added your requested subdomain to JS.ORGs zonefile. But you have to (re-)add a file named ```CNAME``` to your branch with the single line ```react-chart.js.org``` to make it work.
I'll merge after we have "contact".

tannerlinsley commented:
Added 👍
On Sun, Mar 5, 2017 at 11:24 PM Stefan Keim <notifications@github.com>
wrote:

> I've already added your requested subdomain to JS.ORGs zonefile. But you
> have to (re-)add a file named CNAME to your branch with the single line
> react-chart.js.org to make it work.
> I'll merge after we have "contact".
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/js-org/dns.js.org/pull/1108#issuecomment-284312672>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFUmCcF7rriAgUrI8wG48YGLE7j9iVmqks5ri6aEgaJpZM4MTrTb>
> .
>

indus commented:
Welcome https://react-chart.js.org
